### ISSUE TITLE
Update Voyager Apache server CORS to allow PUT methods

### ISSUE THREAD

**Issue by sdellis** [pulibrary/requests#45]
If I'm understanding this correctly, the [RecallPerItem API requires a PUT method](https://developers.exlibrisgroup.com/voyager/apis/RESTfulWebServices/RecallPerItem), but the response I get back from the server is a 405 method not allowed:

```
   response_headers={"date"=>"Wed, 21 Dec 2016 21:46:08 GMT", "server"=>"Apache", "allow"=>"OPTIONS,GET,HEAD,POST,TRACE", "content-length"=>"256", "connection"=>"close", "content-type"=>"text/html; charset=iso-8859-1"},
```

Who can update this?

**Comment by kevinreiss** [comment]
Considering this is a vendor product if we really need this it has to go to Ex Libris. What is the URL are you trying with the PUT request. These APIs are really badly done and I would not be surprised if the above message is incorrect based on past experience with other options. I think the URL for the PUT request literally has to be:

```
http://catalog.princeton.edu:7014/vxws/record/{bibkey}/items/{itemkey}/recall?patron={Patron_key}&patron_homedb=PRINCETON```

along with the XML document. When I did the above with a charged item and my own patron ID I got a different error response about the patron being not allowed to use the database (my account has some sort of borrowing block right now) but it did not reject using that method.
